src/main/python/cfn_sphere/exceptions.py

```diff
@@ -30,9 +43,11 @@ class InvalidEncryptedValueException(CfnSphereException):
     pass
 
 
-class CfnSphereBotoError(CfnSphereException):
+class CfnSphereBotoErrorException(CfnSphereException):
```

Comment:
I would be against this change, because having something named `ErrorException` is just plain ugly. I think we can live with inconsistent naming, some things are `Errors` and others are `Exceptions` -- that is fine by me.